packages/api/README.md

```diff
@@ -88,24 +88,32 @@ are available:
   Lower lever; compatible with most JS engines.
 
 Every `@atproto/oauth-client-*` implementation has a different way to obtain an
-OAuth based API agent instance. Here is an example restoring a previously
-saved session:
+`OAuthSession` instance that can be used to instantiate an `Agent` (from
+`@atproto/api`). Here is an example restoring a previously saved session:
 
 ```typescript
+import { Agent } from '@atproto/api'
 import { OAuthClient } from '@atproto/oauth-client'
 
 const oauthClient = new OAuthClient({
   // ...
 })
 
-const agent = await oauthClient.restore('did:plc:123')
+const oauthSession = await oauthClient.restore('did:plc:123')
+
+// Instantiate the api Agent using an OAuthSession
+const agent = new Agent(oauthSession)
 ```
 
 ### API calls
 
 The agent includes methods for many common operations, including:
 
 ```typescript
+// The DID of the user currently authenticated (or undefined)
+agent.did
+agent.accountDid // Throws if the user is not authenticated
```

Comment:
Interesting! I just noticed this part of the api for the first time.

Small thing: I wonder if this should be `assertDid` instead of `accountDid` to make this clearer?

Reply:
yep that would probably have been a better name -_-

Reply:
I'd recommend we add `assertDid` and deprecate `accountDid` at some point soon.

Reply:
https://github.com/bluesky-social/atproto/commit/bbca17bc5388e0b2af26fb107347c8ab507ee42f